src/schedule.c

```diff
@@ -190,14 +189,14 @@ void destroy_schedule( schedule_t *s )
 
 
 /* See schedule.h for details. */
-char* get_blocked_at_time( schedule_t *s, time_t unixtime )
+char* get_blocked_at_time( schedule_t *s, time_t unixtime, time_t *next_unixtime )
```

Comment:
This function doesn't help get the next_unixtime, so the functionality should live in the function described below instead.  Revert this function.